examples/tv-app/android/java/ContentAppCommandDelegate.cpp

```diff
@@ -81,11 +81,13 @@ void ContentAppCommandDelegate::InvokeCommand(CommandHandlerInterface::HandlerCo
             env->ExceptionDescribe();
             env->ExceptionClear();
             FormatResponseData(handlerContext, "{\"value\":{}}");
-            return;
         }
-        JniUtfString respStr(env, resp);
-        ChipLogProgress(Zcl, "ContentAppCommandDelegate::InvokeCommand got response %s", respStr.c_str());
-        FormatResponseData(handlerContext, respStr.c_str());
+        else
```

Comment:
Previous code was not doing DeleteLocalRef in the exception check case, however new  code does. So the change is a bit more than just "place in a codeblock to fix order of releasing things". Is this intended?

Reply:
Yes. I did not want to add checks for null. The [DeleteLocalRef](https://docs.oracle.com/javase/10/docs/specs/jni/functions.html#deletelocalref) accepts a null and ignores. So I just made the change to call it irrespective.